test/acceptance/tools/resources/config_template.js

```diff
@@ -25,7 +25,7 @@ config.resource = {
         /**
          * Administration port for the proxy.
          */
-        adminPort: 11211
+        adminPort: {{administration_port}}
```

Comment:
Yes, the parameter is set in the pep config by the properties test file. In the function there is a default value (if you want, you can omit the value set properties, and use the default one)

NTC